discretize/BaseMesh.py

```diff
@@ -373,6 +377,46 @@ def copy(self):
         """
         return properties.copy(self)
 
+    axis_u = properties.Vector3(
+        'Vector orientation of u-direction',
+        default='X',
+        length=1
+    )
+    axis_v = properties.Vector3(
+        'Vector orientation of v-direction',
+        default='Y',
+        length=1
+    )
+    axis_w = properties.Vector3(
+        'Vector orientation of w-direction',
+        default='Z',
+        length=1
+    )
+
+    @properties.validator
+    def _validate_orientation(self):
+        """Check if axes are orthogonal"""
+        if not (np.abs(self.axis_u.dot(self.axis_v) < 1e-6) and
+                np.abs(self.axis_v.dot(self.axis_w) < 1e-6) and
+                np.abs(self.axis_w.dot(self.axis_u) < 1e-6)):
+            raise ValueError('axis_u, axis_v, and axis_w must be orthogonal')
+        return True
+
+    @property
+    def is_cartesian(self):
```

Comment:
It is not actually clear to me if this will return False for a cyl mesh at the moment? Would it make sense for the meshes to have a `coordinate_system` property? it can take on the value `cartesian`, `cylindrical`, `spherical`

Reply:
Done in fb5caa7